src/main/java/school/lemon/changerequest/java/Accountant.java

```diff
@@ -0,0 +1,28 @@
+package school.lemon.changerequest.java;
+
+
+/**
+ * Created by lera on 15.12.16.
+ */
+public class Accountant extends Employee {
```

Comment:
As far as I see, your Accountant will use same `calculateSalary` method as `Employee`, but `Employee.calculateSalary` implementation fit only Programmes requirements.
Accountant and Manager should have same logic for `calculateSalary` method.